landing-page/pages/index.js

```diff
@@ -1,22 +1,25 @@
-import Head from "next/head";
-import classes from "./index.module.css";
-import Navbar from "./components/navbar";
+import Head from 'next/head';
+import classes from './index.module.css';
+import Navbar from './components/navbar';
+import Layout from './HOC/Layout';
 export default function Home() {
   return (
-    <div className={classes.home}>
-      <Navbar />
-      <div className={classes.main}>
-        <div className={classes.main_left}>
-          <h1>CoronaVirus Outbreak</h1>
-          <p>The coronavirus outbreak is affecting the lives of millions.</p>
+    <Layout>
+      <div className={classes.home}>
+        <Navbar />
```

Comment:
Shouldn't Navbar be inside the Default Layout?

Reply:
Yes, the Navbar is inside Layout
![image](https://user-images.githubusercontent.com/37709578/83591689-f47de400-a571-11ea-912c-510cbe293626.png)